src/object/task/taskgoto.cpp

```diff
@@ -126,7 +127,7 @@ bool CTaskGoto::EventProcess(const Event &event)
                         if (a || b)
```

Comment:
The debug feature gets a boost in performance when we draw only the active area of the bitmap.
Like this a few lines above:
```diff
-                for (int x = 0; x < m_bmSize; x++)
+                for (int x = m_bmMinX; x <= m_bmMaxX; x++)
                 {
-                    for (int y = 0; y < m_bmSize; y++)
+                    for (int y = m_bmMinY; y <= m_bmMaxY; y++)
```